src/connectionController.ts

```diff
@@ -23,7 +24,11 @@ function getConnectWebviewContent() {
   </html>`;
 }
 
-export default class ConnectionManager {
+export enum DataServiceEventTypes {
+  connectionsDidChange
```

Comment:
SDAM events were talking about at lunch:
- https://github.com/mongodb/node-mongodb-native/blob/master/docs/guide/management/sdam-monitoring.txt
- which you can listen for on ds https://github.com/mongodb-js/data-service/blob/master/lib/native-client.js

Reply:
https://github.com/mongodb/node-mongodb-native/blob/master/test/tools/sdam_viz

Reply:
nice - made a ticket for it: https://jira.mongodb.org/browse/VSCODE-29